src/app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if current_user && user != current_user %> 
+    <% follows = follows?(current_user, user) %>     
```

Comment:
we don't need the first parameter as it's assumed to be current_user as the other helper methods `liked?` and `saved?` .
and by extension we don't need it as a variable, you can use it directly in the next line `link_to_if follows?(user)...`
```suggestion
    <% follows = follows?(user) %>     
```

Reply:
@a7madM we missed this comment ^